### PR TITLE
Add callback url option to fix regression for OAuth2 1.4

### PR DESCRIPTION
**Pull request by gauravtiwari**
* Update old ruby syntax
* Fixes regression in omniauth-oauth2 v1.4.0 by intridea/omniauth-oauth2@85fdbe1 
* Update readme

### CODE REVIEW

**Comment by gauravtiwari** [comment]
@nashby Please review and publish a new version to ruby gems :)

**Comment by gauravtiwari** [comment]
Looks like travis is having a bad day

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## 0.2.1 (September 22, 2015) ##
 
+## Bug fix
+* Fixes regression in omniauth-oauth2 v1.4.0 by [intridea/omniauth-oauth2@85fdbe1](https://github.com/intridea/omniauth-oauth2/commit/85fdbe1)
+
+## 0.2.1 (September 22, 2015) ##
```
Comment (nashby) [inline CHANGELOG.md]: looks like you copy-pasted version as well here :) 
Can you just put that README addition to the `## Unreleased` section?

**Review by nashby — CHANGES_REQUESTED** [review]

**Comment by nashby** [comment]
@gauravtiwari hey, thanks for PR!

**Comment by nashby** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by gauravtiwari** [comment]
Hey, totally forgot about this. Awesome ❤️

**Pull request merged**